maas-schemas/schemas/maas-backend/subscriptions/subscription.json

```diff
@@ -139,6 +139,9 @@
         },
         "unitPrice": {
           "$ref": "#/definitions/price"
+        },
+        "image": {
```

Comment:
Could we use a reference to `core/components/units.json#url` here, rather than string